static/app/views/settings/account/accountEmails.spec.tsx

```diff
@@ -106,7 +106,30 @@ describe('AccountEmails', function () {
     render(<AccountEmails />);
     expect(mock).not.toHaveBeenCalled();
 
-    await userEvent.type(screen.getByRole('textbox'), 'test@example.com{enter}');
+    const mockGet = MockApiClient.addMockResponse({
+      url: ENDPOINT,
+      method: 'GET',
+      statusCode: 200,
+      body: [
+        ...AccountEmailsFixture(),
+        {
+          email: 'test@example.com',
+          isPrimary: false,
+          isVerified: false,
+        },
+      ],
+    });
+
+    const textbox = await screen.findByRole('textbox');
+    const expectedSecondaryEmails = AccountEmailsFixture().length - 1;
+    expect(screen.getAllByLabelText('Remove email')).toHaveLength(
+      expectedSecondaryEmails
+    );
+
+    await userEvent.type(textbox, 'test@example.com{enter}');
+    expect(screen.getAllByLabelText('Remove email')).toHaveLength(
```

Comment:
This might be a little flakey because of the GET request to the API request. 🤔 i wonder if we can tie this to the `mockGet` request you already have? (I guess because that's mocked it's unlikely to be flakey too 🎉)